lib/icon/module.js

```diff
@@ -109,11 +117,15 @@ async function generateIcons (options) {
     options._icons[size] = `${options.targetDir}/icon_${size}.${options.iconHash}.png`
   }
 
+  // Generate _purpose
+  options._purpose = options.purpose ? options.purpose.join(' ') : undefined
```

Comment:
Do we need `_purpose` anywhere else? If not, can we just use `const purpose =  ...`?

Reply:
5638999a09ed341bc1a6533cd84246b06998519d